ENIGMAsystem/SHELL/Graphics_Systems/General/GStiles.h

```diff
@@ -19,6 +19,12 @@
 
 #include "Universal_System/scalar.h"
 
+#include <map>
+#include <vector>
+
+extern std::map<int,int> tile_layer_models;
```

Comment:
why isn't this in a namespace?

Reply:
I don't know. I was just trying to get it compiling first. I actually thought that since `draw_tiles` helper for `screen_redraw` uses no system-specific functions now, it could actually be moved to `GStiles.cpp` too and delete some more duplicate garbage. So then `GStiles.h` really only has to declare `draw_tiles` signature in `namespace enigma`, and not the maps/vectors, to be used by `screen_redraw`. That sound ok?